helmfile/charts/rclone-sync/files/rclone.conf

```diff
@@ -19,3 +19,29 @@ endpoint = {{ .Values.config.destination.s3.regionEndpoint }}
 force_path_style = {{ .Values.config.destination.s3.forcePathStyle }}
 v2_auth = {{.Values.config.destination.s3.v2Auth }}
 {{- end }}
+
+{{- if .Values.config.encrypt.enabled }}
+{{- range .Values.buckets }}
+
+{{ if hasKey . "destination" -}}
```

Comment:
Doesn't they always have destination?

Reply:
right, source and destination is the bucket name :facepalm: I was somehow thinking it was the endpoint to the other S3 service